### PR TITLE
feat: update widgets for usage_unit 

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
**PLEASE SEE 'File changed'**

스크린샷 2023-10-06 오후 5 59 27

스크린샷 2023-10-06 오후 5 59 40

### CODE REVIEW

File: apps/web/src/services/dashboards/widgets/cost-widgets/aws-cloud-front-cost/AWSCloudFrontCostWidget.vue [hunk 1]
```diff
@@ -125,45 +118,49 @@ const state = reactive({
...
-        const tableData: TableData[] = state.data.results.map((d: Data) => {
+        return state.data.results.map((d: Data) => {
             const row: TableData = {
                 [groupBy]: d[groupBy],
             } as TableData;
-            d[`${state.fieldsKey}_sum`]?.forEach((subData: SubData) => {
-                row[subData[USAGE_TYPE_VALUE_KEY]] = subData.value;
-                row.usage_unit = subData.usage_unit;
+            d.value_sum?.forEach((subData: SubData) => {
+                const rowKey = state.fieldsKey === 'usage_quantity' ? `${subData[USAGE_TYPE_VALUE_KEY]}_${subData.usage_unit}` : subData[USAGE_TYPE_VALUE_KEY];
```
Comment (yuda110) [inline apps/web/src/services/dashboards/widgets/cost-widgets/aws-cloud-front-cost/AWSCloudFrontCostWidget.vue:139]: description: ex. `HTTP Requests_Bytes`

File: apps/web/src/services/dashboards/widgets/cost-widgets/aws-cloud-front-cost/AWSCloudFrontCostWidget.vue [hunk 2]
```diff
@@ -178,18 +175,23 @@ const state = reactive({
...
+        const usageTypeValueKeyList = state.data?.results?.[0]?.value_sum.map((d) => d[USAGE_TYPE_VALUE_KEY]) ?? [];
+        return uniq(usageTypeValueKeyList).length !== usageTypeValueKeyList.length;
+    }),
+    usageUnitMap: computed<Record<string, string|null>>(() => {
+        const result: Record<string, string|null> = {};
+        state.data?.results?.[0]?.value_sum?.forEach((d: SubData) => {
+            result[d[USAGE_TYPE_VALUE_KEY]] = d.usage_unit;
+        });
+        return result;
+    }),
```
Comment (yuda110) [inline apps/web/src/services/dashboards/widgets/cost-widgets/aws-cloud-front-cost/AWSCloudFrontCostWidget.vue:194]: I have doubts about whether it's reasonable to retrieve the 0th item here.

여기서 0번째 아이템을 가져오는 것이 과연 합리적인가 의문이 듭니다🥲

**Review by WANZARGEN — APPROVED** [review]
I checked everything and add the commit to this PR. :)
Thanks for your hard work!

**Review by piggggggggy — APPROVED** [review]
LGTM!

**Pull request merged**